README.md

```diff
@@ -0,0 +1,94 @@
+Ecosystem Edge NVIDIA-CI - Golang Automation CI
+=======
+# NVIDIA-CI
+
+## Overview
+The [nvidia-ci](https://github.com/rh-ecosystem-edge/nvidia-ci) is an automation/CI framework to test NVIDIA operators, starting with the GPU Operator.
+This project is based on golang + [ginkgo](https://onsi.github.io/ginkgo) framework.
+
+### Project requirements
+* golang v1.20.x
+* ginkgo v2.x
```

Comment:
How about we refer people to the `go.mod` file for getting the requirements?
This part of the README is going to be outdated very fast and probably forgotten.

Reply:
Good idea.  Updated